roles/engineblock/templates/engine.conf.j2

```diff
@@ -48,4 +48,14 @@ Listen {{ apache_app_listen_address.engine }}:{{ loadbalancing.engine.port }}
     Include ssl_backend.conf
     {% endif %}
 
+    <IfModule mod_expires.c>
```

Comment:
This means that if you don't enable the module it will silently just not work as intended. Would it be more predictable to remove the guard so Apache will alert you if you forgot to enable the module?

Reply:
Good point, should we advertise the dependency on this module as a requirement somewhere in docs?